src/LensPlatformClient.ts

```diff
@@ -6,9 +6,8 @@ import { PermissionsService } from "./PermissionsService";
 import { InvitationService } from "./InvitationService";
 import { PlanService } from "./PlanService";
 import axios, { AxiosRequestConfig } from "axios";
-
+import pino from "pino";
 import decode from "jwt-decode";
-import _console from "./helpers/_console";
```

Comment:
do we still use `_console` somewhere?

Reply:
in ext there was a legacy version, but it's a different instance from this.